src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,110 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactNode } from 'react';
+import { merge } from '@utilities/merge';
+import { ContainerHTMLElement } from 'src/types/elements';
+import { Box } from '@components/Box';
+
+export type FlexDirection = 'row' | 'column' | 'row-reverse' | 'column-reverse';
+export type FlexWrap = 'nowrap' | 'wrap' | 'wrap-reverse';
+export type FlexJustify = 'start' | 'end' | 'center' | 'between' | 'around' | 'evenly';
+export type FlexAlignItems = 'start' | 'end' | 'center' | 'baseline' | 'stretch';
+export type FlexAlignContent =
+    | 'start'
+    | 'end'
+    | 'center'
+    | 'between'
+    | 'around'
+    | 'evenly'
+    | 'baseline'
+    | 'stretch'
+    | 'normal';
+
+export type FlexProps = {
```

Comment:
We could add `padding` and `margin` to the `Flex` component the same way we are using on `Container`